content/operate/kubernetes/release-notes/7-2-4-releases/7-2-4-12-10-24.md

```diff
@@ -0,0 +1,147 @@
+| Azure AKS |  |  | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |
+| Google GKE | <span title="X icon">&#x274c;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |
+| Rancher 2.6 | <span title="X icon">&#x274c;</span> | <span title="X icon">&#x274c;</span> |  |  |  |  |
+| Rancher RKE2 | <span title="Deprecation warning">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| VMware TKG 2.1 | <span title="Deprecation warning">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> |  |  |  |  |
+| VMware TKG 2.2 | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |  |
+| VMware TKG 2.3 |  | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| **OpenShift version** | **4.10** | **4.11** | **4.12** | **4.13** | **4.14** |  |
+|  | <span title="X icon">&#x274c;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |
+| **VMware TKGI version** | **1.14** | **1.15** |  |  |  |  |
+|  | <span title="X icon">&#x274c;</span> | <span title="Supported">&#x2705;</span> |  |  |  |  |
+
+## Downloads
+
+- **Redis Enterprise**: `redislabs/redis:7.2.4-117`
+- **Operator**: `redislabs/operator:7.2.4-12`
+- **Services Rigger**: `redislabs/k8s-controller:7.2.4-12.`
+
+#### OpenShift images
+
+- **Redis Enterprise**: `registry.connect.redhat.com/redislabs/redis-enterprise:7.2.4-117.rhel8-openshift`
+    (or `redislabs/redis-enterprise:7.2.4-117.rhel7-openshift` if upgrading from RHEL 7)
+- **Operator**: `registry.connect.redhat.com/redislabs/redis-enterprise-operator:7.2.4-12`
+- **Services Rigger**: `registry.connect.redhat.com/redislabs/services-manager:7.2.4-12`
+
+#### OLM bundle
+
+**Redis Enterprise operator bundle** : `v7.2.4-12.21`
+
+## Known limitations
+
+The limitations below are specific to the Redis Enterprise for Kubernetes product. To view limitations for the supported Redis Enterprise Software version, see the [7.2.4 release notes]({{< relref "/operate/rs/release-notes/rs-7-2-4-releases#known-limitations" >}}).
+
+### Existing limitations
+
+- **When modifying the database suffix for an Active-Active database, while the service-rigger is in a terminating state, the services-rigger will delete and create the ingress or route resources in a loop (RED-107687)** Wait until the services- rigger pod has finished to terminate it.
```

Comment:
Is it correct to have both "service-rigger" and "services-rigger"? Also, space in "services- rigger".